edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/BrokerProperties.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.Json;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// DTO that is used to deserialize MQTT Broker Config from EdgeHub twin
+    /// into <see cref="BrokerConfig" />.
+    /// </summary>
+    public class BrokerProperties
```

Comment:
Why do we need this and BrokerConfig separately? The Option?

Reply:
Yeah, I couldn't make it work with Option<T> so I split json model and domain model. I guess at some point we can fix `OptionConverter`, but this will be a separate task.